www/source/partials/docs/_bp-kubernetes.html.md.erb

```diff
@@ -1,17 +1,29 @@
 # <a name="kubernetes" id="kubernetes" data-magellan-target="kubernetes">Kubernetes</a>
 [Kubernetes](http://kubernetes.io/) is an open source container cluster manager that is available as a stand-alone platform or embedded in several distributed platforms including [Google's Container Engine](https://cloud.google.com/container-engine/), [Tectonic](https://tectonic.com/) by [CoreOS](https://coreos.com/), and [OpenShift](https://openshift.com/) by [RedHat](https://redhat.com). Habitat and Kubernetes are complementary: Kubernetes focuses on providing a platform for deployment, scaling, and operations of application containers across clusters of hosts while Habitat manages the build pipeline and lifecycle of those application containers.
 
-## Habitat Kubernetes Operator
```

Comment:
Nit: Think we agreed to call it Habitat operator at some point.

Reply:
Yeah but thinking of it, Habitat Operator makes more sense IMO. Also, it's a title so we could also argue that all non-articles/pronouns are capitalized (although the rest of the document is not consistent in this).

Don't want to bikeshed though so if you feel strongly about this I can change.

Reply:
Nope, no strong opinions at all.